rocks-db/src/columns/editions.rs

```diff
@@ -33,6 +35,35 @@ impl TypedColumn for TokenMetadataEdition {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, Default, Hash, PartialEq, Eq)]
+pub struct EditionIndexKey {
+    pub pub_key: Pubkey,
+    pub edition: u64,
+}
+
+impl TypedColumn for TokenMetadataEditionParentIndex {
+    type KeyType = EditionIndexKey;
+    type ValueType = Self;
+    const NAME: &'static str = "TOKEN_METADATA_EDITION_PARENT_INDEX"; // Name of the column family
+
+    fn encode_key(search_key: EditionIndexKey) -> Vec<u8> {
+        encode_pubkey_u64(search_key.pub_key, search_key.edition)
+    }
+
+    fn decode_key(encoded_key: Vec<u8>) -> Result<Self::KeyType> {
+        let (pub_key, edition) = decode_pubkey_u64(encoded_key)?;
+        Ok(EditionIndexKey { pub_key, edition })
+    }
+
+    fn decode(bytes: &[u8]) -> Result<Self::ValueType> {
+        serde_cbor::from_slice(bytes).map_err(|e| StorageError::Common(e.to_string()))
+    }
+
+    fn encode(v: &Self::ValueType) -> Result<Vec<u8>> {
+        serde_cbor::to_vec(&v).map_err(|e| StorageError::Common(e.to_string()))
+    }
```

Comment:
why do we use CBOR as opposed to bincode?

Reply:
Indeed. afaik Oleksandr got rid of `Cbor` in favour of `bincode`.

Reply:
I took the current solution that was used in the editions.rs

Reply:
We still use cbor for editions, raw black and parameters